packages/url_launcher/url_launcher_linux/linux/test/url_launcher_linux_test.cc

```diff
@@ -39,6 +39,31 @@ TEST(UrlLauncherPlugin, CanLaunchFailureUnhandled) {
                              expected));
 }
 
+TEST(UrlLauncherPlugin, CanLaunchFileSuccess) {
+  g_autoptr(FlValue) args = fl_value_new_map();
+  fl_value_set_string_take(args, "url", fl_value_new_string("file:/"));
```

Comment:
Shouldn't this be `file:///`? I'm surprised this works.

Reply:
This is proper mimimal representation of the file scheme URI pointin to local linux root folder:
> The minimal representation of a local file with no authority field and an absolute path that begins with a slash "/"

[rfc8089#appendix-B](https://datatracker.ietf.org/doc/html/rfc8089#appendix-B)